source/server/api/v1/routes.js

```diff
@@ -14,6 +15,21 @@ import {
 
 const router = express()
 
+router.get(
+  '/auth/google',
+  passport.authenticate( 'google', { scope:
+  [
+    'https://www.googleapis.com/auth/plus.login',
+    'https://www.googleapis.com/auth/plus.profile.emails.read'
+  ]
+}))
+
+router.get( '/google/auth/callback',
+  passport.authenticate( 'google', { failureRedirect: '/' } ),
+  ( request, response ) => {
+    response.status(200).json({ message: 'shit failed' })
```

Comment:
Status code 200 for failure? Also, be descriptive with the message so the future humans will not cry when they get errors.